packages/flutter_tools/gradle/flutter.gradle

```diff
@@ -18,6 +18,7 @@ import org.gradle.api.tasks.bundling.Jar
 
 buildscript {
     repositories {
+        google()
         jcenter()
         maven {
```

Comment:
The entire `maven {...}` should be removed.  It looks like our problem is that jcenter is being used first, and we want to use google.

I think it's fine to use `google()` at this point, as we're not using gradle versions old enough to not know about it, but having the redudancy in here doesn't make sense to me.